packages/l/libsdl3/xmake.lua

```diff
@@ -0,0 +1,132 @@
+package("libsdl3")
+    set_homepage("https://www.libsdl.org/")
+    set_description("Simple DirectMedia Layer")
+    set_license("zlib")
+
+    if is_plat("mingw") and is_subhost("msys") then
+        add_extsources("pacman::SDL3")
+    elseif is_plat("linux") then
+        add_extsources("pacman::sdl3", "apt::libsdl3-dev")
+    elseif is_plat("macosx") then
+        add_extsources("brew::sdl3")
+    end
+
+    add_urls("https://www.libsdl.org/release/SDL3-$(version).zip",
+             "https://github.com/libsdl-org/SDL/releases/download/release-$(version)/SDL3-$(version).zip", { alias = "archive" })
+    add_urls("https://github.com/libsdl-org/SDL.git", { alias = "github" })
+
+    add_versions("archive:3.2.0", "58d8adc7068d38923f918e0bdaa9c4948f93d9ba204fe4de8cc6eaaf77ad6f82")
+    add_versions("archive:3.2.0", "abe7114fa42edcc8097856787fa5d37f256d97e365b71368b60764fe7c10e4f8")
+
+    add_versions("github:3.2.2", "release-3.2.2")
+    add_versions("github:3.2.0", "release-3.2.0")
+
+    add_deps("cmake", "egl-headers", "opengl-headers")
+
+    if is_plat("linux", "bsd", "cross") then
+        add_configs("x11", {description = "Enables X11 support", default = true, type = "boolean"})
+        add_configs("wayland", {description = "Enables Wayland support", default = nil, type = "boolean"})
```

Comment:
if default is nil, we need not set it.

Reply:
yeah that's just to make it explicit and not let other developer think it's missing